### DISCUSSION TITLE
list out of range (error)

### DISCUSSION THREAD

**Discussion by fafafauzan** [teamalgoritma/community#763] [Q&A]
![image](https://github.com/teamalgoritma/community/assets/94038423/439184e0-72a0-4708-a69e-6265ca1575f6)

**Comment by arkanayudhistira** [comment]
Dear Pak @fafafauzan,

Apakah boleh ditunjukkan code yang bapak gunakan ketika mendefinisikan `row_length`? karena error tersebut kemungkinan disebabkan oleh nilai jumlah baris (`row_length`) yang lebih besar daripada jumlah baris yang sebenarnya pada website tersebut.

**Comment by divakartika** [comment]
Mohon izin menutup diskusi ini karena belum ada respon dari penanya selama 2 pekan, semoga dapat dipahami.
